webpack.config.dev.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Should this config file going into the `@wordpress/scripts` package?

Reply:
I think we can skip `@wordpress/scripts` for the time being. The existing Gutenberg config is completely independent and way more complex 😅 See: https://github.com/WordPress/gutenberg/blob/master/webpack.config.js. My bet is that we should move all custom logic related to React Fast Refresh there first.